ember.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
   extends: [
     require.resolve('./'),
   ],
+  root: true,
```

Comment:
what does this do?

Reply:
why do we need it?

Reply:
We don't. It just ensures that an ember eslint config doesn't affect other repositories besides itself. It could be beneficial in the future but is not currently.  

That is all.